cloud-etl/README.md

```diff
@@ -0,0 +1,121 @@
+![image](../images/confluent-logo-300-2.png)
+
+# Overview
+
+This demo showcases a cloud ETL leveraging all fully-managed services on [Confluent Cloud](https://confluent.cloud?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cloud-etl)
+A source connector reads data from an AWS Kinesis stream into Confluent Cloud, a Confluent KSQL application processes that data, and then a sink connector writes the output data into cloud storage in the provider of your choice.
+
+![image](images/topology.jpg)
+
+This enables you to:
+
+* Build business applications on a full event streaming platform
+* Span multiple cloud providers (AWS, GCP, Azure) and on-prem datacenters
+* Use Kafka to aggregate data in single source of truth
+* Harness the power of [KSQL](https://www.confluent.io/product/ksql/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cloud-etl)
+
+# End-to-end Streaming ETL
+
+This demo showcases an entire end-to-end cloud ETL deployment, built for 100% cloud services:
+
+* Kinesis source connector: reads from a Kinesis stream and writes the data to a Kafka topic in Confluent Cloud
+* ksqlDB: streaming SQL engine that enables real-time data processing against Kafka
+* GCS or S3 sink connector: pushes data from Kafka topics to cloud storage
+
+| Component                  | Consumes From              | Produces To             |
+|----------------------------|----------------------------|-------------------------|
+| Kinesis source connector   | Kinesis stream `demo-logs` | Kafka topic `eventLogs` |
+| KSQL                       | `eventLogs`                | KSQL streams and tables ([ksql.commands](ksql.commands)) |
+| GCS/S3/Blob sink connector | KSQL tables `COUNT_PER_SOURCE`, `SUM_PER_SOURCE` | GCS/S3/Blob         |
+
+# Warning
```

Comment:
If the customer starts in Kinesis but sinks to GCS or Azure Blob, the data will have to be egressed from AWS, which means we'll pay the highest data transfer cost. I believe our current pricing is a blended price assuming most data transfer will be intra-zone or intra-region. Net: people running this demo could cost us more than we charge them. 

Pinged Nandish from PMM in Slack to confirm: https://confluent.slack.com/archives/C9QRB4HJ7/p1581446961216300

Fallback could be limiting this example for Kinesis -> S3 (which would make it less cool because the cross-cloud story is awesome)

Reply:
@bechols good discussion point, let's follow up in Slack.